etherman/etherman.go

```diff
@@ -104,6 +104,8 @@ type Client struct {
 	GasProviders externalGasProviders
 
 	auth *bind.TransactOpts // nil in case of read-only client
+
+	cfg Config
```

Comment:
I don't understand this change.

Reply:
Oups well spotted @tclemos. I was testing to log some values from the config. Removed 309a01585b0f4e335731e599ae7dd83905ea4880